pyproject.toml

```diff
@@ -29,10 +29,11 @@ classifiers = [
 dependencies = [
   'astropy>=5.3.3',
   'numpy>=1.18.0',
-  'sunpy>=5.0.1',
-  'spacepy>=0.5.0',
+  'sunpy[all]>=5.0.1',
+  'spacepy>=0.4.1',
```

Comment:
Why do we need to revert the `spacepy` version?

Reply:
Good catch, that was definitely a typo